third-party/dependencies

```diff
@@ -1 +1,2 @@
 https://github.com/cesanta/mongoose/archive/refs/tags/7.9.tar.gz
+https://github.com/nlohmann/json/releases/download/v3.11.2/json.hpp
```

Comment:
Missing end of line here.

Reply:
Ok, I actually changed this in the next PR that is on top of this one: https://github.com/cartesi/machine-emulator/pull/152